Dec2022Demo/test_S2.jl

```diff
@@ -0,0 +1,134 @@
+using Catlab.CategoricalAlgebra
+using Catlab.Present, Catlab.Theories
+using AlgebraicPetri
+using AlgebraicPetri: Graph
+
+include("LibForOct2022Demo.jl");
+
+# Type system model
+types′ = LabelledPetriNet([:Pop],
+    :infect=>((:Pop, :Pop)=>(:Pop, :Pop)),
+    :disease=>(:Pop=>:Pop),
+    :strata=>(:Pop=>:Pop))
+types = map(types′, Name=name->nothing)
+
+# Parts of type system for ease of reference
+s, = parts(types′, :S)
+t_interact, t_disease, t_strata = parts(types′, :T)
+i_interact1, i_interact2, i_disease, i_strata = parts(types′, :I)
+o_interact1, o_interact2, o_disease, o_strata = parts(types′, :O);
+
+# SIRD augmented model
+SIRD_aug = LabelledPetriNet([:S, :I, :R, :D],
```

Comment:
Don't we not need these additional id transitions because of the stratification API?